compiler/control/OMROptions.cpp

```diff
@@ -122,6 +122,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
       "narrower than 32-bits (boolean, byte, char, short) are in-range",
       SET_OPTION_BIT(TR_AllowVPRangeNarrowingBasedOnDeclaredType), "F" },
    {"alwaysFatalAssert",       "I\tAlways execute fatal assertion for testing purposes",           SET_OPTION_BIT(TR_AlwaysFatalAssert), "F"},
+   {"alwaysProfileDuringStartup", "R\t enforce iprofiler during startup phase", SET_OPTION_BIT(TR_AlwaysProfileDuringStartup), "F", NOT_IN_SUBSET},
```

Comment:
Maybe this options should be renamed "alwaysIProfileDuringStartupPhase" to indicate that we are talking about interpreter profiler and not JIT profiler. Also, in description I would add this option overrides the other one `noIProfilerDuringStartupPhase`